.drone.yml

```diff
@@ -4,9 +4,9 @@ build:
     image: golang:1.8-alpine
     commands:
       - apk -qq --no-cache add git
-      - go get github.com/Masterminds/glide
+      - go get github.com/golang/dep
```

Comment:
The right command to install dep is `go get github.com/golang/dep/cmd/dep` (WHYYYYYYY?).

Fixing this line should fix the build.